src/DynamoCoreWpf/ViewModels/Core/CustomColorPickerViewModel.cs

```diff
@@ -0,0 +1,218 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using System.Windows.Media;
+using Dynamo.Core;
+using Xceed.Wpf.Toolkit;
+
+namespace Dynamo.ViewModels
+{
+    internal class CustomColorItem : ColorItem, INotifyPropertyChanged
+    {
+        public event PropertyChangedEventHandler PropertyChanged;
```

Comment:
Comments for public properties please

Reply:
@QilongTang comments added in the next commit: https://github.com/DynamoDS/Dynamo/pull/13794/commits/a49d970a769a23938d84abc67836a231e4de443d